bookstore/bookstore.py

```diff
@@ -1,29 +1,56 @@
 def create_bookstore(name):
-    pass
+    authors = []
```

Comment:
Instead of doing `authors = []` and `books = []` you can put that right in your dictionary definition:
```
bookstore = { 'name': name,
              'authors': [],
              'books': [] }
```